[ENG-27351] new fields limit (#298)
aziontech/docs: src/content/docs/en/pages/api-graphql/limits/limits.mdx

## Query rows limit

You can query up to *10,000 rows per query*. The **limit** field in your query controls this restriction, and you can set it as you desire as long as it doesn’t surpass the limit. If you surpass it, the API will return an error response message informing the issue.

---

> ## Selected fields limit Real-Time Metrics GraphQL API
> 

Author: ```suggestion
```

> The maximum value of selected fields for a dataset is of *35 fields per query* in the Real-Time Metrics GQL. If you surpass the fields' limit, the API will return an error response message informing the issue.

Reviewer: ```suggestion
The maximum value of selected fields for a dataset is of *35 fields per query* in the Real-Time Metrics GraphQL API. If you surpass the limit of fields, the API will return an error response message informing the issue.
```

Mesmo comment da Mari aqui em baixo.

Author: é pra ser GQL, mesmo :D

---

## Selected fields limit Real-Time Events GraphQL API

> The maximum value of selected fields for a dataset is of *10 fields per query* in the Real-Time Events GQL. If you surpass the fields' limit, the API will return an error response message informing the issue. If you need to analyze more fields, create different queries.

Reviewer 2: ```suggestion
The maximum value of selected fields for a dataset is of *10 fields per query* in the Real-Time Events GraphQL API. If you surpass the fields' limit, the API will return an error response message informing the issue. If you need to analyze more fields, create different queries.
```
